website/data/commands-navigation.js

```diff
@@ -0,0 +1,78 @@
+export default [
```

Comment:
I don't know which file has the content, but noticed that Key Concepts is spelled as Key Conceps in the sidebar in the preview here under https://deploy-preview-8506--consul-docs-preview.netlify.app/docs/install/concepts